Tests/HummingbirdTests/ExtensionTests.swift

```diff
@@ -14,92 +14,93 @@
 
 @testable import Hummingbird
 import XCTest
+/*
```

Comment:
This code can also be removed now that it's commented

Reply:
done